tests/Behat/Foundry.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Behat support for Foundry
+
+    As a developer writing tests with Behat, I want to be able to
+    use Foundry for creating test data. Foundry needs to be configured,
+    started and stopped accordingly. The database has to be reset before
+    scenarios are run.
+
```

Comment:
PS might be worth adding a test for "example tested" syntax.
https://behat.org/en/latest/user_guide/writing_scenarios.html#scenario-outlines

Associated events for subscriber: ExampleTested::BEFORE, ExampleTested::AFTER